spec/classes/server/conf_spec.rb

```diff
@@ -190,6 +192,8 @@
 
         context 'with useprivilegeseparation' do
           let(:facts) { os_facts.merge( { :openssh_version => '6.6' } ) }
+          let(:pre_condition){ "service {'sshd':}" }
```

Comment:
Why was this added?

Reply:
The test as it was was failing on Puppet 5